params/config.go

```diff
@@ -425,6 +426,59 @@ func (c *ChainConfig) Description() string {
 	return banner
 }
 
+// BlobConfig specifies the target and max blobs per block for the associated
+// fork.
+type BlobConfig struct {
+	Target uint64 `json:"target"`
+	Max    uint64 `json:"max"`
+}
+
+// BlobScheduleConfig determines target and max number of blobs allow per fork.
+type BlobScheduleConfig struct {
+	Cancun *BlobConfig `json:"cancun,omitempty"`
+	Prague *BlobConfig `json:"prague,omitempty"`
+}
+
+// LatestTargetBlobsPerBlock returns the target blobs per block associated with
+// requested time.
+func (c *ChainConfig) LatestTargetBlobsPerBlock(time uint64) uint64 {
```

Comment:
not sure about the `Latest` prefix here

Reply:
would you rather it removed or a different prefix?

Reply:
I would say let's remove it.